src/components/EditFeatures.jsx

```diff
@@ -1,24 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import ModalLayout from './ModalLayout';
+import CTA from './CTA';
 import ShareImg from '../images/share.svg';
 import PenImg from '../images/pen.svg';
 import DeleteImg from '../images/trash.svg';
 import styles from './EditFeatures.module.css';
 
-export default function EditFeatures() {
+export default function EditFeatures({ folderId, folderName }) {
+  const [editFolderNameModalOpen, setEditFolderNameModalOpen] = useState(false);
+  const [newFolderName, setNewFolderName] = useState('');
+
+  const [deleteFolderModalOpen, setDeleteFolderModalOpen] = useState(false);
+
+  const handleEditFolderName = async (e) => {
+    e.preventDefault();
+    if (!newFolderName) return;
+    const editFolderName = async () => {
+      await fetch(`https://bootcamp-api.codeit.kr/api/folders/${folderId}`, {
```

Comment:
동일하게 사용되는 BASE_URL은 따로 파일을 만들어서 불러오는것이 좋을 것 같아요!
대부분 이런 경우에는 env 파일을 이용해서 BASE_URL을 만들고 필요한 api 주소만 사용해 줍니다

`REACT_APP_BASE_URL = https://bootcamp-api.codeit.kr/api/`

예를 들어 위와 같이 만들어 두었다면

```suggestion
      await fetch(`process.env.REACT_APP_BASE_URL/folders/${folderId}`, {
```

이런 방식으로 작성이 가능하겠습니다.